apps/public-docsite-v9/src/AccessibilityScenarios/Accordion1.stories.tsx

```diff
@@ -0,0 +1,78 @@
+import * as React from 'react';
+
+// https://github.com/microsoft/fluentui/pull/18695#issuecomment-868432982
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
```

Comment:
why is this needed ? this was solved some time ago. can you pls remove those ts-ignores ?

Reply:
just copied over, not new code